src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -621,6 +621,65 @@ g.test('simple_blend_constant,blend_constant_non_inherited')
     t.expectOK(result);
   });
 
+g.test('color_write_mask,blending_disabled')
```

Comment:
Not fully sure if this test can be here. Please comment if this file is unsuitable to test the write mask.

Reply:
I'd say make a small new file for this, but it's more convenient to use the helpers defined here. How about renaming the file from `blending` to `color_target_state`, and adding "blending" to the name of any unclear tests?

Reply:
ok, I rename `blending.spec.ts` file to `color_target_state.spec.ts`, and then add `blending` prefix to `format` and `GPUBlendComponent`. Thanks for the suggestion!